server.js

```diff
@@ -15,12 +15,15 @@ var express = require('express'),
 // generate a new express app and call it 'app'
 var app = express();
 
+var db = require('./models');
 // serve static files in public
 app.use(express.static('public'));
 
 // body parser config to accept our datatypes
 app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
If you are going to be testing with postman, you'll want to make sure to configure bodyParser to also work with JSON.  Look back at yesterday's code for an example.